source/agora/node/Ledger.d

```diff
@@ -125,37 +125,72 @@ public class Ledger
             throw new Exception("Genesis block loaded from disk is " ~
                 "different from the one in the config file");
 
-        // need to regenerate the UTXO set, starting from the genesis block
+        // need to rebuild the UTXO set and Validator set,
+        // starting from the genesis block
         if (this.utxo_set.length == 0)
         {
             Block block;
```

Comment:
Please move this to line 143. It's not used outside the `foreach` loop

Reply:
I forgot to move it.